monitor/requirements.txt

```diff
@@ -0,0 +1,3 @@
+fastapi==0.85.1
```

Comment:
should monitor be a lambda function or a server. 
If it is a lambda function than you do not need fast api and uvicorn.

If it is a server on ec2 you dont have a handler like in server-monitor.py